docs/source/keywords.rst

```diff
@@ -272,6 +272,10 @@ Architecture Keywords
       - Description/info
       - Default
 
+    * * ``Kokkos_ARCH_NATIVE``
+      * Let compiler detect the architecture of the machine compiling on
```

Comment:
```suggestion
      * Optimize for the local CPU architecture
```
Would be more inline with with the way other architectures in `BUILD.md` are treated here.